packages/schemas/tables/applications.sql

```diff
@@ -8,6 +8,7 @@ create table applications (
   type application_type not null,
   oidc_client_metadata jsonb /* @use OidcClientMetadata */ not null,
   custom_client_metadata jsonb /* @use CustomClientMetadata */ not null default '{}'::jsonb,
+  role_names jsonb /* @use RoleNames */ not null default '[]'::jsonb,
```

Comment:
I created LOG-4220 and will add a migration file for this.

Reply:
Cool, I can do it